applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/harmonic_analysis_strategy.hpp

```diff
@@ -507,6 +532,38 @@ class HarmonicAnalysisStrategy
 
         KRATOS_CATCH("")
     }
+    
+    /**
+     * @brief This method provides the defaults parameters to avoid conflicts between the different constructors
+     * @return The default parameters
+     */
+    Parameters GetDefaultParameters() const override
+    {
+        Parameters default_parameters = Parameters(R"(
+        {
+            "name"                          : "harmonic_analysis_strategy",
+            "builder_and_solver_settings"   : {},
+            "linear_solver_settings"        : {},
+            "scheme_settings"               : {},
+            "harmonic_analysis_settings"    : {
```

Comment:
where are u taking the defaults from?

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_harmonic_analysis_solver.py